game/game.css

```diff
@@ -51,129 +51,129 @@ table{
     background-size: cover;
 }
 
-.map_wall--image{
+.map--wall--img{
     background-image: url(tiles/wall.png);
     background-repeat: no-repeat;
     background-position: center;
     background-size: cover;
 }
 
-.map_wall--image--lamp{
+.map--wall--img--lamp{
     background-image: url(tiles//wall1.png);
     background-repeat: no-repeat;
     background-position: center;
     background-size: cover;
 }
-.map_wall--image--door{
+.map--wall--img--door{
```

Comment:
map_wall--image jest zrobione w konwencji BEM, wiec czemu to zmieniac? 
lepeij reszte zmieniec do BEMa tez

Reply:
corrected to:
map__tile--wall
map__tile--floor